### ISSUE TITLE
Error after upgrade to hacs-hubitat 0.7.1 && hubitat 2.3.1.132

### ISSUE THREAD

**Issue by yougotborked** [jason0x43/hacs-hubitat#164]
```
This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/hubitat/hub.py:242
Integration: Hubitat (documentation, issues)
First occurred: 12:33:39 PM (3 occurrences)
Last logged: 12:36:10 PM

Error setting up entry Hubitat (6ccc5096) for hubitat
Error setting up entry Hubitat zwave (6ccc5096) for hubitat
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 335, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/hubitat/__init__.py", line 37, in async_setup_entry
    if not await hub.async_setup():
  File "/config/custom_components/hubitat/hub.py", line 242, in async_setup
    await self._hub.start()
  File "/usr/local/lib/python3.9/site-packages/hubitatmaker/hub.py", line 189, in start
    await self._start_server()
  File "/usr/local/lib/python3.9/site-packages/hubitatmaker/hub.py", line 459, in _start_server
    await self.set_event_url(self.event_url)
  File "/usr/local/lib/python3.9/site-packages/hubitatmaker/hub.py", line 236, in set_event_url
    await self._api_request(f"postURL/{url}")
  File "/usr/local/lib/python3.9/site-packages/hubitatmaker/hub.py", line 420, in _api_request
    raise RequestError(resp)
hubitatmaker.error.RequestError: GET http://10.1.1.50/apps/api/8/postURL/http:%2F%2F10.1.1.52:41137?access_token=6ccc5096-c731-4ed3-ad71-dbe3ecda4173 - [500] Server Error
```

I tried deleting and re-adding the integration, same issue

It looks like it is doing a GET to a postURL?
both my c4 and c7 are experiencing the issue.

**Comment by yougotborked** [comment]
Actually I see this in the hubitat log
```
[app:8](http://10.1.1.50/logs#pastapp8)2022-04-01 13:05:02.022 [error](http://10.1.1.50/installedapp/configure/8)groovy.lang.MissingMethodException: No signature of method: java.lang.String.call() is applicable for argument types: (com.hubitat.hub.domain.Location, java.lang.String) values: [basement zwave, eventHandler]
Possible solutions: wait(), any(), trim(), collect(), size(), split() on line 524 (method setPostURL)
[app:8](http://10.1.1.50/logs#pastapp8)2022-04-01 13:05:01.530 [debug](http://10.1.1.50/installedapp/configure/8)url=http://10.1.1.52:42195
[app:8](http://10.1.1.50/logs#pastapp8)2022-04-01 13:05:01.526 [debug](http://10.1.1.50/installedapp/configure/8)setPostURL called
```

I did also upgrade hubitat to  2.3.1.132  at the same time on both the c7 and c4. Mabye the c7 has a bug the c4 doesn't have?

**Comment by yougotborked** [comment]
Turning this off fixes the issue? but then I don't get any status/mode management as I'm sure you know.
![image](https://user-images.githubusercontent.com/1314122/161326046-8d84f4e3-4679-44ed-97ff-4e0539d3a50c.png)

**Comment by jason0x43** [comment]
It looks like there was a [bug](https://community.hubitat.com/t/maker-api-issues-after-latest-update/91699/20) in 2.3.130. They should have a fix out in the next update.

**Comment by jason0x43** [comment]
I guess I'll be holding off on that for a bit longer (I'm still on 2.3.0.124). 😄

**Comment by yougotborked** [comment]
Looks like they released 2.3.1.134 today which seems t have fixed the issue
